src/lib/Fullscreen.js

```diff
@@ -99,6 +99,13 @@ class Fullscreen extends EventEmitter {
 
         if (this.isSupported()) {
             if (this.isFullscreen()) {
+                // Focus on the fullscreen element so keyboard
+                // events are triggered without an extra click
+                // If el has target property, then it is an Event
+                // otherwise it is a HTMLElement
+                const element = el.target || el;
```

Comment:
Wouldnt we want to do this for both cases when enter=true?
If so, then it should be in the if block below, and not here